.gitignore

```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
```

Comment:
`.env.local` doesn't work for me and throws the following error. Once I renamed it to just `.env` the error went away.

```
yarn db:migrate:deploy
yarn run v1.22.19
$ prisma migrate deploy
Prisma schema loaded from schema.prisma
Datasource "db": PostgreSQL database

Error: Prisma schema validation - (get-config wasm)
Error code: P1012
error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:10
   |
 9 |   provider = "postgresql"
10 |   url      = env("DATABASE_URL")
   |

Validation Error Count: 1
[Context: getConfig]

Prisma CLI Version : 4.12.0
error Command failed with exit code 1.
```

Reply:
This is strange. NextJS specifically recommends to use .env.local https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables as opposed to .env